### PR TITLE
Create component page for mysensors based cover

### PR DESCRIPTION
**Pull request by OttoWinter**
See home-assistant/home-assistant#3512

### CODE REVIEW

**Comment by Streberpower** [comment]
Currently, the example sketch for MySensors covers is written for MySensors v2.0, whereas all other [home-assistant.github.io](https://home-assistant.io/) MySensors pages are written for v1.5. In MySensors-land, v2.0 seems to be where all the cool new stuff is happening right now. I could, however, easily port the example to v1.5, if that was required.

File: source/_components/cover.mysensors.markdown [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+  send(stopMessage.set(state == IDLE));
+  send(statusMessage.set(status));
+}
+
+void setup() {
+  pinMode(COVER_UP_SENSOR_PIN, INPUT);
+  pinMode(COVER_DOWN_SENSOR_PIN, INPUT);
+
+  // Send initial values.
+  sendState();
```
Comment (MartinHjelmare) [inline source/_components/cover.mysensors.markdown]: In mysensors 2.0.0 this has to be done in the loop, due to registration on gateway required before device can send values. This has changed in dev branch of mysensors, but that's not released yet.

File: source/_components/cover.mysensors.markdown [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+    sendState();
+    Serial.println("Stopping cover.");
+
+    // Actuators will be switched off in loop().
+  }
+}
+```
+
+[main component]: /components/mysensors/
+[serial api]: https://www.mysensors.org/download/serial_api_15
```
Comment (MartinHjelmare) [inline source/_components/cover.mysensors.markdown]: This link is not working anymore, cause they removed that site. Use API 2.0 instead:
https://www.mysensors.org/download/serial_api_20

**Comment by MartinHjelmare** [comment]
I think it's good to add example sketches for version 2.0 from now on and rather complement the existing sketches with ones for version 2.0. So no need to change that.

**Comment by Streberpower** [comment]
Feel free to change the code, I'm on my phone right now and it will take some time for me to get back to my development environment.

**Comment by OttoWinter** [comment]
@MartinHjelmare I just realised that I have set `ha_release` in the header section to `0.29`. This should probably be updated when the [PR](home-assistant/home-assistant#3512) is merged.

**Comment by MartinHjelmare** [comment]
Yes!

File: source/_components/cover.mysensors.markdown [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+title: "MySensors Cover"
+description: "Instructions how to integrate MySensors covers into Home Assistant."
+date: 2016-09-25 11:30 +0100
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: mysensors.png
+ha_category: Cover
+ha_release: 0.29
```
Comment (MartinHjelmare) [inline source/_components/cover.mysensors.markdown]: Feature has been merged so change `ha_release` to 0.30.

**Pull request merged**